package-lock.json

```diff
@@ -2206,6 +2206,22 @@
                 "readable-stream": "^2.0.6"
```

Comment:
You haven't updated any node modules, so package-lock.json should not be modified. These are probably changes to your local environment that you made in previous revisions and didn't revert

Reply:
In the future, at least on this repo, if you're not modifying package.json, you should always discard changes to package-lock.json before committing. Now it's a bit more annoying to restore after you've already committed the changes.